sdks/python/apache_beam/runners/interactive/display/pcoll_visualization_test.py

```diff
@@ -88,44 +87,40 @@ def test_dynamic_plotting_return_handle(self):
     h.stop()
 
   @patch('apache_beam.runners.interactive.display.pcoll_visualization'
-         '.PCollectionVisualization.display_facets')
+         '.PCollectionVisualization._display_dive')
+  @patch('apache_beam.runners.interactive.display.pcoll_visualization'
+         '.PCollectionVisualization._display_overview')
+  @patch('apache_beam.runners.interactive.display.pcoll_visualization'
+         '.PCollectionVisualization._display_dataframe')
   def test_dynamic_plotting_update_same_display(self,
-                                                mocked_display_facets):
+                                                mocked_display_dataframe,
```

Comment:
Are we verifying that the same display is being updated? If so consider s/update/updates, if not consider a name that communicates the expected behavior.

Reply:
Yes, we are verifying that the same display is being updated.
Renaming the test to `test_dynamic_plotting_updates_same_display`.

Reply:
SG - once you push the commit and tests pass we can merge this.

Reply:
Thanks!